test/parallel/test-http2-client-stream-destroy-before-connect.js

```diff
@@ -7,11 +7,6 @@ const NGHTTP2_INTERNAL_ERROR = h2.constants.NGHTTP2_INTERNAL_ERROR;
 
 const server = h2.createServer();
 
-// we use the lower-level API here
-server.on('stream', common.mustCall((stream) => {
-  stream.on('aborted', common.mustCall());
-}));
-
```

Comment:
Will need to look at this again later. On windows, nothing makes it to the server, on Linux/Mac, it does. So this may or may not be called based on what platform we're on